src/cli/bruker2dicom

```diff
@@ -58,13 +61,34 @@ def main():
     logging.getLogger().setLevel(verbosity.upper())
     
     function = arguments.pop("function")
+    source = arguments.pop("source")
+    
+    #create tmp directory in the current location
+    extractdir = tempfile.mkdtemp("","",".")
```

Comment:
No, the root directory is system-dependent: it's not mentioned in the `mkdtemp` doc, but it's explicit in the [mkstemp doc](https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp).